packages/updatePrescriptionStatus/tests/testHandler.test.ts

```diff
@@ -69,6 +69,33 @@ describe("Integration tests for updatePrescriptionStatus handler", () => {
     expect(response.statusCode).toEqual(201)
     expect(JSON.parse(response.body)).toEqual(responseSingleItem)
 
+    expect(expectedItems.input.TransactItems[0].Put.Item.RepeatNo).toEqual({S: 1})
```

Comment:
This test shouldn't know about RepeatNo. That should all go into the new test below.